### DISCUSSION TITLE
Footer links color

### DISCUSSION THREAD

**Discussion by Shalini469717** [karmacircleCommunity/KarmaCircle#655] [General]
Since the theme of the website seems to be orange, I think it would be good if we can make the links, icons or report button to orange too.

![image](https://user-images.githubusercontent.com/99305545/235345797-fa1ff159-18cb-4d8a-9943-ebd9b4ffb948.png)

**Comment by tamalCodes** [comment]
Yep, that is true - apparently, we've got someone working on this in #623  - so let's see if these changes are made 🔥

**Reply by Shalini469717** [reply]
Yeah sure
